pkg/reconciler/knativeserving/ingress/gloo.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"context"
+
+	mf "github.com/manifestival/manifestival"
+	"knative.dev/operator/pkg/apis/operator/v1alpha1"
+)
+
+var glooFilter = ingressFilter("gloo")
+
+func glooTransformers(ctx context.Context, instance *v1alpha1.KnativeServing) []mf.Transformer {
+	return nil
```

Comment:
Interesting you return `nil` here and not an empty `[]mf.Transformer{}`...

Reply:
If it is nil, then the length of the array is 0.

Reply:
Idiomatic Golang allows for transparent use of `nil` slices. Their len is 0 and you can `range` over them no problem. Allows for saving the allocations that returning an empty slice forces.